LinkRelay/plugin.py

```diff
@@ -317,10 +338,12 @@ def send(s):
                 tail = [cont + ' ' + s[i:i+remainingLength] for i in
                         range(allowedLength, len(s), remainingLength)]
                 for s in [head] + tail:
-                    msg = f(relay.targetChannel, s)
+                    s = '\x01ACTION {}\x01'.format(s) if args.get('draft/relaymsg') and args.get('action') else s
```

Comment:
```
if args.get('draft/relaymsg') and args.get('action'):
    s = '\x01ACTION %s\x01' %s 
```

More readable + doesn't need a line wrap + using percent-substitution for consistency